File: 3/03035.py
news = []
R, C, ZR, ZC = map(int, input().split())
for i in range(R):
    news.append(input())

for i in range(R):
    for ii in range(ZR):
        for j in range(C):
            for jj in range(ZC):
                print(news[i][j], end='')
        print()

File: 3/03035.cpp
#include <iostream>
using namespace std;

char news[50][50];

int main() {
	int R, C, ZR, ZC; cin >> R >> C >> ZR >> ZC;
	for (int i = 0; i < R; i++)
		for (int j = 0; j < C; j++)
			cin >> news[i][j];

	for (int i = 0; i < R; i++) {
		for (int ii = 0; ii < ZR; ii++) {
			for (int j = 0; j < C; j++)
				for (int jj = 0; jj < ZC; jj++)
					cout << news[i][j];
			cout << endl;
		}
	}
}
